src/@types/document.ts

```diff
@@ -1,3 +1,7 @@
+import { OpenAttestationDocument as OpenAttestationDocumentV2 } from "../__generated__/schemaV2";
```

Comment:
Would you like to add a comment that it's generated by quicktype and can see in postinstall? I was wondering where this came from

Reply:
done =)